ofxdump/ofxdump.cpp

```diff
@@ -1201,12 +1202,15 @@ int main (int argc, char *argv[])
         cout << "file: " << args_info.inputs[i] << endl ;
       }
     }
-    return libofx_proc_file(libofx_context, args_info.inputs[0], file_format);
+    int rc = libofx_proc_file(libofx_context, args_info.inputs[0], file_format);
```

Comment:
sorry, that's wrong: The return value here should be returned by the `main` function. Storing it in a locally declared variable will discard it, which is not what we want. You must write `rc = libofx_proc_file...` instead (without the declaration).

Reply:
Fixed.